core/commands/publish.go

```diff
@@ -89,7 +91,8 @@ Publish an <ipfs-path> to another public key (not implemented):
 		}
 
 		// TODO n.Keychain.Get(name).PrivKey
-		output, err := publish(n, n.PrivateKey, p)
+		// TODO(cryptix): is req.Context().Context a child of n.Context()?
```

Comment:
I think that it is part of the nodes 'context group' somehow. Although I dont claim to understand context groups, or how they different from processes